CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+#### Contributing to a Manim Community Fork - General guidelines
+
+As this is a Fork of [Manim from 3b1b](https://github.com/3b1b/manim), contributing can be a bit confusing. So here is a small guide on it. 
+
+1. First, make a fork of either this Repo or the Repo from which this was Forked. ( 3b1b Repository)
+2. Then clone the repository you forked by the command below.
+
+ ```sh
+   git clone <your-fork-url>
+   cd manim
+ ```
+
+3. Now that you have cloned the Repo, before making any changes, you have to enter the commands below to contribute to Manim Community Fork.
+
+   ```sh
+   git remote add fork https://github.com/ManimCommunity/manim.git
```

Comment:
I'd rather call this repo "upstream" instead of "fork", since we are recommending that the user also has their own fork, so "fork" is ambiguous here.